sugarcrm/customer.py

```diff
@@ -0,0 +1,36 @@
+import sugarcrm
+from sugarcrm.client import APIClient
+
+
+class Customer:
+    def __init__(self, logger=sugarcrm.logger):
+        self.logger = logger
+        self.roles = set()
+        self.api_client = APIClient()
```

Comment:
we could inject the api_client as argument which would allow for injecting a custom one in tests, instead of using the setter